src/fellows.ts

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+import { ApiPromise, WsProvider } from "@polkadot/api";
+
+import { ActionLogger } from "./github/types";
+
+type FellowData = { address: string; rank: number };
+
+export const fetchAllFellows = async (
+  logger: ActionLogger,
+): Promise<Map<string, number>> => {
+  let api: ApiPromise;
+  logger.debug("Connecting to collective parachain");
+  // we connect to the collective rpc node
+  const wsProvider = new WsProvider(
+    "wss://polkadot-collectives-rpc.polkadot.io",
```

Comment:
do we want later to add more chains besides polkadot/collectives?

Reply:
I would say no for now.

If we want, we can create an input type and set it there, but let's not over engineer it right now (as the modification would be simple if we actually need it)